src/main/java/com/aiykr/iquais/service/impl/UserServiceImpl.java

```diff
@@ -174,22 +179,45 @@ public Response<UserResponseDTO> getStudentById(String id) {
     }
 
     /**
-     * Retrieves a list of all users from the database.
+     * Retrieves a list of all users from the database by page.
```

Comment:
The JAVADOC for this method should also mention the parameters (page, size, sortBy, sortOrder).